dashboard/http_server_agent.py

```diff
@@ -53,12 +53,16 @@ async def start(self, modules):
 
         self.runner = aiohttp.web.AppRunner(app)
         await self.runner.setup()
-        site = aiohttp.web.TCPSite(
-            self.runner,
-            "127.0.0.1" if self.ip == "127.0.0.1" else "0.0.0.0",
-            self.listen_port,
-        )
-        await site.start()
+        try:
+            site = aiohttp.web.TCPSite(
+                self.runner,
+                "127.0.0.1" if self.ip == "127.0.0.1" else "0.0.0.0",
+                self.listen_port,
+            )
+            await site.start()
+        except OSError as e:
+            logger.error("Agent port has been used. Fail to start agent.")
```

Comment:
Can we also include the port number in this error message? Something like `f"Agent port #{ray_constants.DEFAULT_DASHBOARD_AGENT_LISTEN_PORT} already in use. Failed to start agent. Ensure port #{ray_constants.DEFAULT_DASHBOARD_AGENT_LISTEN_PORT} is available, and then try again."`